domain_access/src/DomainAccessManager.php

```diff
@@ -66,6 +67,7 @@ public function getAccessValues(EntityInterface $entity, $field_name = DOMAIN_AC
    * @inheritdoc
    */
   public function getAllValue(EntityInterface $entity) {
+    /** @var FieldableEntityInterface $entity */
```

Comment:
We should probably change the type hint then and see if any of the tests explode.